sdk/datafactory/Azure.ResourceManager.DataFactory/src/autorest.md

```diff
@@ -43,32 +49,77 @@ rename-rules:
   URI: Uri
   MWS: Mws
   Etag: ETag|etag
+  Db: DB|db
 
 rename-mapping:
```

Comment:
In `rename-mapping`, I saw that sometimes we prepend `Factory` but sometimes `DataFactory`, can we align these renaming, or do you think it's better to keep this change?

Reply:
Consolidate to `Factory` except the one `DataFactory` model